.github/workflows/docker_launcher_release.yml

```diff
@@ -31,6 +31,11 @@ jobs:
         with:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
+      
+      - name: Install skopeo
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y skopeo
```

Comment:
Should we pin the version here somewhere to avoid time bombs?

Reply:
as we discussed, version pinning for ubuntu packages might create a time bomb by itself, so leaving it as is